src/components/UserCard.jsx

```diff
@@ -1,17 +1,17 @@
-import profile from "../assets/images/profile-img.svg";
-import messages from "../assets/icons/messages-icon.svg";
+import profile from "../assets/images/profile.svg";
+import messages from "../assets/icons/messages.svg";
 function UserCard() {
   return (
     <>
       <div className="border-grayscale-40 flex h-[168px] w-[154px] flex-col gap-[32px] rounded-[16px] border p-[16px]">
         <div className="flex flex-col gap-[12px]">
           <img className="h-[48px] w-[48px]" src={profile}></img>
```

Comment:
```suggestion
          <img className="h-48 w-48" src={profile}></img>
```
--spacing 설정이 있으므로 간략하게 숫자로만 적는 것이 좋을 것 같습니다